app/jobs/articles/async_bust_job.rb

```diff
@@ -0,0 +1,11 @@
+module Articles
+  class AsyncBustJob < ApplicationJob
+    queue_as :articles_async_bust
```

Comment:
I'd call this `articles_bust_cache` to mimic the name `comments_bust_cache` of the other job